src/components/CompareResults/RunValues.tsx

```diff
@@ -47,7 +47,7 @@ function RunValues(props: RunValuesProps) {
         </div>
       ) : (
         <div>
-          <b>{name}:</b> {formatNumber(median)} {measurementUnit}
+          <b>{name}:</b> {formatNumber(avg)} {measurementUnit}
```

Comment:
you also need to do it above Neta

Reply:
Oh okay I didn't realize they should be the same. Thanks for pointing that out